.github/workflows/conformance-gateway-api.yaml

```diff
@@ -167,7 +167,9 @@ jobs:
             --gateway-class cilium \
             --supported-features ReferenceGrant,TLSRoute,HTTPRouteQueryParamMatching,HTTPRouteMethodMatching,RouteDestinationPortMatching,GatewayClassObservedGenerationBump,HTTPResponseHeaderModification \
             -test.run "TestConformance" \
-            -test.skip "TestConformance/HTTPRouteListenerHostnameMatching" # Enable once #24217 is fixed
+            -test.skip "TestConformance/HTTPRouteListenerHostnameMatching|TestConformance/HTTPRouteRedirectHostAndStatus" 
```

Comment:
can you update the gatewayapi version in this file as well ? 

https://github.com/cilium/cilium/blob/2f11767a22547f8fac98d4c2b36b326524db8934/.github/workflows/conformance-gateway-api.yaml#L28